src/BlogContent/SyntheticMixedBenchmark/README.md

```diff
@@ -0,0 +1,80 @@
+We continuously improve the performance of the Redshift service. One workload that we use is the Cloud Data Warehouse Mixed Read/Write Benchmark Derived from TPC-H (http://www.tpc.org/tpch/default.asp).
+
+Note: The TPC Benchmark and TPC-H are trademarks of the Transaction Processing Performance Council (http://www.tpc.org). The Cloud Data Warehouse Mixed Read/Write Benchmark is derived from the TPC-H Benchmark and as such is not comparable to published TPC-H results.
+
+    "The TPC Benchmark (TPC-H) is a decision support benchmark. It consists of a suite of business oriented ad-hoc queries and concurrent data modifications. The queries and the data populating the database have been chosen to have broad industry-wide relevance while maintaining a sufficient degree of ease of implementation. 
+	This benchmark illustrates decision support systems that
+
+        Examine large volumes of data;
+        Execute queries with a high degree of complexity;
+        Give answers to critical business questions."
+
+The main intention to create this benchmark is to mimic a real world mixed workload scenario, where some applications continuously read from datamart set of tables, Analysts run adhoc queries throughout the day, ingestions are happening against the larger data warehouse set of tables on a periodic manner. Also there are some pre-defined set of reports which run on a schedule. In this derived benchmark, we have broken down these 22 queries into 3 categories based on the run timings. The short ones we marked as DASHBOARD queries. The longest ones marked as DATASCIENCE and medium ones are categorized as REPORT. The corresponding queries would also be submitted with different speed (configurable) and concurrency. All these queries are submitted in an infinite loop with a finite wait between them. The wait time is introduced to mimic a scenario where data analysts process the query output data locally for next 30 mins before submitting another query. The concurrency is to represent the number of concurrent users. In real world, different group of users also uses different data sets. For example, API calls the datamart tables for quick lookup whereas analysts generally work against the larger data warehouse set of tables. Common practice that Data ingestion happens against the larger dataset frequently.
+
+With the attached scripts, DASHBOARD queries are submitted as 16 concurrent users with 2 secs wait, REPORT queries in 6 parallel streams with 15 mins wait, 4 parallel streams of DATASCIENCE queries with 30 mins wait. We have the mix dataset of both tpc-h 100g and tpc-h 3t. The smaller tpc-h 100g represents a datamart and the larger tpc-h 3t is the main data warehouse. REPORT and DATASCIENCE queries read from the larger tpc-h 3t dataset, against which the COPY loads are going in every hour. For the copy we load the tpc-h 100g data files on top of the tpc-h 3t data tables.
+
```

Comment:
Add a line, For more information, please see <blog link>  If we don't yet have the blog link we can add this later.